packages/react-jsx-highcharts/src/components/Series/Series.js

```diff
@@ -80,7 +81,7 @@ const Series = memo(
       let doRedraw = false;
       // Using setData is more performant than update
       if (isDataEqual(data, prevProps.data) === false) {
-        series.setData(data, false);
+        series.setData(data, false, undefined, updatePoints);
```

Comment:
Think we should support the third arg (animation) here via a prop too? Which would default to `undefined`

Reply:
[The "animation" can be passed to series as option.](https://api.highcharts.com/highcharts/series.line.animation)

So different name would be needed.

Reply:
Good spot!

Reply:
Maybe `animated` as a prop name?